src/dune_engine/process.mli

```diff
@@ -163,6 +163,16 @@ val run_capture_zero_separated
   -> string list
   -> 'a Fiber.t
 
+(** [run_external_in_out] differs from the other [run] functions in the
+    followings ways:
+
+    - The process group ID is inherited by the parent process rather than
+    creating a new one.
+
+    - The input and output file descriptors are the standard ones.
+
+    This version is intended for running external processes at the end of a
+    build such as the ones spawned with "dune exec". *)
 val run_external_in_out
```

Comment:
The name is rather mysterious. How about something like `run_inherit_std_in_out`?